libraries/SPI/SPI.cpp

```diff
@@ -403,16 +426,24 @@ void ArduinoSPI::configSpiSci(arduino::SPISettings const & settings)
   uint32_t smr  = R_SCI0_SMR_CM_Msk;
 
   /* Configure CPHA setting. */
-  spmr |= (uint32_t) clk_phase << 7;
+  uint32_t mask = (ONE_BIT << SCI_CKPH_POS);
+  spmr &= ~(uint32_t)mask; // reset bit 0
```

Comment:
Is this comment `// reset bit 0` true? It looks to me like its copy and paste from the baud rate generator clock divider configuration.